twentyseventeen/sepsis-quiz.php

```diff
@@ -29,10 +28,6 @@
             <div class="card-container">
                   <div class="question">Question</div>
                   <div class="choices">
-                        <div>Choice A</div>
```

Comment:
I would delete the whole example or update it so it reflects what we currently output.